tests/storage/datavolume.go

```diff
@@ -1416,3 +1471,11 @@ func newRandomVMWithCloneDataVolume(sourceNamespace, sourceName, targetNamespace
 	libstorage.AddDataVolumeTemplate(vm, dataVolume)
 	return vm
 }
+
+func havePrintableStatus(status v1.VirtualMachinePrintableStatus) gomegatypes.GomegaMatcher {
```

Comment:
Nice. Do you think we could reuse this?

Reply:
Why no.
Do you think we should move it to `tests/framework/matcher/???` ?

Reply:
[See](https://github.com/kubevirt/kubevirt/blob/main/tests/vm_test.go#L2085-L2091). It would be great to unify.

Reply:
done